lib/modules.nix

```diff
@@ -87,6 +84,16 @@ rec {
       result = { inherit options config; };
     in result;
 
+
+ /* Filter disabled modules. Modules can be disabled allowing
+    their implementation to be replaced. */
+ filterModules = modules:
+   let
+     moduleKey = m: if isString m then toString ../nixos/modules + "/" + m else toString m;
```

Comment:
I just found [`specialArgs.modulesPath`](https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/eval-config.nix#L52). If we used that other module sets would get the same behaviour if they define define it there.